docs/concepts/Security-Best-Practices.md

```diff
@@ -0,0 +1,166 @@
+
+For more information about Dependabot alerts & security updates, [see the following documentation](https://docs.github.com/en/github/managing-security-vulnerabilities/about-alerts-for-vulnerable-dependencies).
+
+### NuGet feeds
+
+Packages can come from different feeds. To ensure you are secure, knowing what feed your packages are coming from is a best practice. One such best practice is the use of a single feed. You can accomplish this by using multiple upstream source feeds to bring your packages into a single feed.
+
+For more information to secure your package feeds, see [3 Ways to Mitigate Risk When Using Private Package Feeds](https://azure.microsoft.com/en-us/resources/3-ways-to-mitigate-risk-using-private-package-feeds/).
+
+### Client trust policies
+
+There are policies that you can opt-into in which you require the packages you use to be signed. This allows you to trust a package author, as long as it is author signed, or trust a package if it is owned by a specific user or account that is repository signed by NuGet.org.
+
+To configure client trust policies, [see the following documentation](../consume-packages/installing-signed-packages.md).
+
+### Lock files
+
+Lock files store the hash of your package’s content. If the content hash of a package you want to install matches with the lock file, it will ensure package repeatability.
+
+To enable lock files, [see the following documentation](../consume-packages/package-references-in-project-files#locking-dependencies).
+
+## Monitor your supply chain
+
+### Publish to NuGet.org
+
+NuGet.org serves as a central repository to over 200,000 unique packages. Whenever you publish a package, NuGet.org will go through numerous validations and indexing that can benefit you in the long term. These can include scanning the package for viruses, [providing a repository signature](../reference/signed-packages-reference.md) on the package, and even protecting your package ID so only you can push updates to it.
+
+To learn more about the benefits of publishing on NuGet.org, see [Package validation and indexing](../nuget-org/publish-a-package.md#package-validation-and-indexing).
+
+### GitHub secret scanning
+
+GitHub scans repositories for NuGet API keys to prevent fraudulent uses of secrets that were accidentally committed. 
+
+To learn more about secret scanning, see [About secret scanning](https://docs.github.com/en/github/administering-a-repository/about-secret-scanning).
+
+### Author Package Signing
+
+[Author signing](../reference/signed-packages-reference.md) allows a package author to stamp their identity on a package and for a consumer to verify it came from you. This protects you against content tampering and serves as a single source of truth about the origin of the package and the package authenticity. 
```

Comment:
Author signing provides security benefits when it's combined with Client Policies. We should state that.

Reply:
We can link back to https://review.docs.microsoft.com/en-us/nuget/concepts/security-best-practices?branch=pr-en-us-2309#client-trust-policies which talks about it.

Reply:
Added a commit.